### PR TITLE
Sync on current 8.2 release

### PR DESCRIPTION
**Pull request by Nambrok**
Koji build (scratch, v8.2-v-linstor-testing): https://koji.xcp-ng.org/taskinfo?taskID=62478

### CODE REVIEW

File: SOURCES/stormi.patch
Comment (stormi) [inline SOURCES/stormi.patch:1]: The removal of this useless patch (oops) should be done first on the 8.2 branch. It's not part of the sync.

File: SPECS/sm.spec [hunk 1]
```diff
@@ -690,20 +676,17 @@ cp -r htmlcov %{buildroot}/htmlcov
...
 - Add 0178-fix-linstor-check-if-resource-is-tiebreaker-62.patch
-
-* Fri Jun 28 2024 Ronan Abhamon <ronan.abhamon@vates.fr> 2.30.8-12.1.0.linstor.3
 - Add 0176-fix-LinstorSR-open-non-leaf-volumes-in-RO-mode-creat.patch
-
-* Mon Jun 17 2024 Ronan Abhamon <ronan.abhamon@vates.fr> 2.30.8-12.1.0.linstor.2
 - Reload automatically multipathd config after each update
-
-* Wed Jun 12 2024 Ronan Abhamon <ronan.abhamon@vates.fr> 2.30.8-12.1.0.linstor.1
 - Add "Provides": sm-linstor (necessary for the "Requires" of xcp-ng-linstor)
```
Comment (Wescoeur) [inline SPECS/sm.spec:684]: Are you sure what you're doing here? :smile:

**Comment by stormi** [comment]
Will need another rebase once #54 is merged.

**Comment by Nambrok** [comment]
It's not a rebase per say, just a sync of the changes by hand. With a format-patch from the same version of sm as the 8.2 one and adding the patches of linstor on top. It is mergeable by itself.

**Review by Wescoeur — APPROVED** [review]

**Comment by Nambrok** [comment]
Fixed the changelog.

**Review by stormi — APPROVED** [review]

**Comment by Nambrok** [comment]
Koji build (v8.2-v-linstor-testing): https://koji.xcp-ng.org/taskinfo?taskID=62484

**Pull request merged**